core/src/main/java/com/pierfrancescosoffritti/androidyoutubeplayer/core/player/views/YouTubePlayerView.kt

```diff
@@ -200,4 +226,16 @@ class YouTubePlayerView(context: Context, attrs: AttributeSet? = null, defStyleA
 
     fun removeFullScreenListener(fullScreenListener: YouTubePlayerFullScreenListener): Boolean =
             fullScreenHelper.removeFullScreenListener(fullScreenListener)
+
+
+    /*function which sets OntouchEvent Listener*/
+    fun setOnTouchListener(onTouchEvent: OnTouchEvent) {
+        if (this.onTouchEvent == null) {
+            this.onTouchEvent = onTouchEvent
+        }
+    }
+
+    interface OnTouchEvent {
```

Comment:
Can't we just use a framework's listener? Like `OnTouchListener`?

Reply:
I have tried, but I can not get any click call back in my view that why I have created my own.